examples/bond/assets/buyback-lsig.py

```diff
@@ -0,0 +1,42 @@
+from pyteal import *
+
+def buyback_lsig():
+    # check no rekeying, close remainder to, asset close to
+    common_fields = And(
+        Gtxn[0].rekey_to() == Global.zero_address(),
+        Gtxn[1].rekey_to() == Global.zero_address(),
+        Gtxn[2].rekey_to() == Global.zero_address(),
+        Gtxn[0].close_remainder_to() == Global.zero_address(),
+        Gtxn[1].close_remainder_to() == Global.zero_address(),
+        Gtxn[2].close_remainder_to() == Global.zero_address(),
+        Gtxn[1].asset_close_to() == Global.zero_address(),
+    )
+
+    # verify that buyer deposits required algos 
```

Comment:
is this comment right? I don't see verifying deposit amount here.

Reply:
thanks, this should be moved to stateful contract